extensions/wikia/FacebookClient/FacebookClient.class.php

```diff
@@ -281,6 +281,19 @@ public function getWikiaUser( $fbId = null ) {
 			return null;
 		}
 
```

Comment:
This process leaves behind mapping rows with App ID == 0.  We can never quite be sure when we can delete those because we don't know what other apps user's if these old style FB IDs might be connected to.  There is logging in ```migrateMapping``` however, and we should see calls to this dwindle to zero over time.  At some point we can decide we've done enough and delete this code and the rows with App ID == 0.  If there are users who still haven't upgraded a row for an app they use, they'll just have to re-connect.